src/Sylius/Bundle/CoreBundle/Remover/ReviewerReviewsRemover.php

```diff
@@ -34,6 +34,9 @@ public function removeReviewerReviews(ReviewerInterface $author): void
         $reviewSubjectsToRecalculate = [];
 
         foreach ($this->reviewRepository->findBy(['author' => $author]) as $review) {
+            if (!$review instanceof ReviewInterface) {
```

Comment:
Shouldn't it be assert? If `reviewRepository` suddenly returns non-reviews object, then we shouldn't continue execution.

Reply:
Yes!